osf_tests/test_framework_utils.py

```diff
@@ -1,52 +0,0 @@
-import pytest
-
-from modularodm import Q
-
-from tests.base import DbTestCase
-from osf_tests import factories
-
-from framework.database import get_or_http_error, autoload
-from framework.exceptions import HTTPError
-
-from osf.models import Node
-
-
-class FrameworkUtilsTestCase(DbTestCase):
```

Comment:
This is totally a mistake on my part I think -- meant to undo this and add to ticket for final pass of removing odm. Sorry bout that.